{"name": "test_login_with_valid_email_and_invalid_password", "status": "passed", "start": 1734967981620, "stop": 1734967982880, "uuid": "b41f4bab-92f1-4862-b347-be0ef74130d3", "historyId": "a87129e8bc488f2c2b04474f72c047f1", "testCaseId": "a87129e8bc488f2c2b04474f72c047f1", "fullName": "tests.test_login.TestLogin#test_login_with_valid_email_and_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "EPINHYDW04A4"}, {"name": "thread", "value": "2156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}